src/dswx_sar/filter_SAR.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 from scipy import signal
 
-from dswx_sar_util import Constants
+from .dswx_sar_util import Constants
```

Comment:
Thank you for correcting this. The relative path is fine, but would you be able to use the absolute path?
```suggestion
from dswx_sar.dswx_sar_util import Constants
```